src/deltacode/test_utils.py

```diff
@@ -46,8 +46,6 @@ def run_scan_click(options, monkeypatch=None, test_mode=True, expected_rc=0, env
     from click.testing import CliRunner
     from deltacode import cli
 
-    options = add_windows_extra_timeout(options)
-
```

Comment:
Currently we do not have the support for adding the cli option --timeout, we may require it for the Appevyor windows CI as it timesout after 120 sec.